wildfly/keycloak-saml-adapter/README.md

```diff
@@ -0,0 +1,7 @@
+# Intersmash Applications - WildFly SAML Adapter + Keycloak
+
+A WildFly/JBoss EAP 8.x application which is provisioned by the `wildfly-maven-plugin` and that uses the
+keycloak-saml-adapter-galleon-pack feature pack to secure resources by authenticating users through SAMP single-sign-on, provided by a 
+Keycloak service.
```

Comment:
Should we mention that the SAML client is registered automatically? I mean, to avoid users go and look for hiding things, as _we know_ it can happen... :-)